packages/deployment/app/cumulus_api.template.yml

```diff
@@ -200,7 +200,7 @@ Resources:
       {{else}}
       Role: {{../parent.iams.lambdaProcessingRoleArn}}
     {{/if}}
-      Runtime: {{# if this.runtime}}{{this.runtime}}{{else}}nodejs6.10{{/if}}
+      Runtime: {{# if this.runtime}}{{this.runtime}}{{else}}nodejs8.10{{/if}}
```

Comment:
oh, thanks for catching this. I only updated the node version on the main template and not on the nested ones.